routesrv/polling.go

```diff
@@ -98,19 +92,20 @@ func (p *poller) poll(wg *sync.WaitGroup) {
 			span.SetTag("error", true)
 			span.LogKV(
 				"event", "error",
-				"message", msg,
+				"message", LogRoutesEmpty,
 			)
 		case routesCount > 0:
-			routesBytes, initialized = p.b.formatAndSet(routes)
+			routesBytes, initialized, updated := p.b.formatAndSet(routes)
 			logger := log.WithFields(log.Fields{"count": routesCount, "bytes": routesBytes})
 			if initialized {
 				logger.Info(LogRoutesInitialized)
 				span.SetTag("routes.initialized", true)
 				routesInitialized.SetToCurrentTime()
-			} else {
+			}
+			if updated {
```

Comment:
should we set SpanTag for `routes.updated`?

Reply:
Makes sense, thank you.
Added in the #2531